### PR TITLE
Feed individual tokens in subword learner

### PR DESCRIPTION
**Pull request by guillaumekln**

### CODE REVIEW

File: nmtwizard/preprocess/consumer.py [hunk 1]
```diff
@@ -16,57 +16,72 @@ def __call__(self, tu_batch):
...
+            # The subword learner needs to be aware of the tokenizer annotations
+            # to properly ignore them.
+            learner_info = tokenizer.make_subword_learner(
+                subword_config,
+                result_dir,
+                tokenizer=tokenizer.build_tokenizer(tokenization_config))
+            self._subword_info[side] = learner_info
+            self._subword_learners[side] = learner_info['learner']
+
+        self._source_learner = self._subword_learners.get('source')
```
Comment (natsegal) [inline nmtwizard/preprocess/consumer.py:56]: Why do we need these extra references ?

**Comment by natsegal** [comment]
Looks good to me

File: nmtwizard/preprocess/consumer.py (hunk 1 above)
Comment (guillaumekln) [reply nmtwizard/preprocess/consumer.py:56]: They save repetitive hash table lookups in the `__call__` method.

Comment (natsegal) [reply nmtwizard/preprocess/consumer.py:56]: OK, thanks for explanation!

**Pull request merged**